<?xml version='1.0' encoding='UTF-8'?>
<rss xmlns:atom="http://www.w3.org/2005/Atom" xmlns:content="http://purl.org/rss/1.0/modules/content/" version="2.0">
  <channel>
    <title>CVE Feed for RDFlib -- pyrdfa3</title>
    <link>https://raw.githubusercontent.com/deepseas/cvelistV5monitor/main/feeds/rdflib/pyrdfa3.rss</link>
    <description>The latest CVEs for RDFlib -- pyrdfa3</description>
    <docs>http://www.rssboard.org/rss-specification</docs>
    <generator>python-feedgen</generator>
    <lastBuildDate>Sat, 03 Aug 2024 23:23:21 +0000</lastBuildDate>
    <ttl>60</ttl>
    <item>
      <title>CVE-2022-4396|2024-08-03T01:41:44.749Z -- rdflib -- pyrdfa3
</title>
      <link>https://www.cve.org/CVERecord?id=CVE-2022-4396</link>
      <description>A vulnerability was found in RDFlib pyrdfa3 and classified as problematic. This issue affects the function _get_option of the file pyRdfa/__init__.py. The manipulation leads to cross site scripting. The attack may be initiated remotely. The name of the patch is ffd1d62dd50d5f4190013b39cedcdfbd81f3ce3e. It is recommended to apply a patch to fix this issue. The identifier VDB-215249 was assigned to this vulnerability. NOTE: This vulnerability only affects products that are no longer supported by the maintainer.</description>
      <guid isPermaLink="false">CVE-2022-4396|2024-08-03T01:41:44.749Z</guid>
      <pubDate>Sat, 10 Dec 2022 00:00:00 +0000</pubDate>
    </item>
  </channel>
</rss>
